stubs/fpdf2/fpdf/enums.pyi

```diff
@@ -8,15 +8,15 @@ class SignatureFlag(IntEnum):
     SIGNATURES_EXIST = 1
     APPEND_ONLY = 2
 
-class CoerciveEnum(Enum):
+class CoerciveEnum(Enum):  # type: ignore[misc]  # Enum with no members
```

Comment:
That might be a false positive? An enum with no members is OK, it makes an abstract class.

Reply:
Yeah; mypy is flagging them with an error like this:

```
error: Detected enum "fpdf.enums.CoerciveEnum" in a type stub with zero members. There is a chance this is due to a recent change in the semantics of enum membership. If so, use `member = value` to mark an enum member, instead of `member: type`  [misc]
```

I didn't check whether there's a better way to silence that.